memor/session.py

```diff
@@ -154,16 +154,67 @@ def add_message(self,
             self._messages_status.insert(index, status)
         self._date_modified = get_time_utc()
 
-    def remove_message(self, index: int) -> None:
+    def get_message_by_index(self, index: Union[int, slice]) -> Union[Prompt, Response]:
         """
-        Remove a message from the session object.
+        Get a message from the session object by index.
+
+        :param index: index
+        """
+        return self._messages[index]
+
+    def get_message_by_id(self, message_id: str) -> Union[Prompt, Response]:
+        """
+        Get a message from the session object by message id.
+
+        :param message_id: message id
+        """
+        for index, message in enumerate(self._messages):
+            if message.id == message_id:
+                return self.get_message_by_index(index=index)
+
+    def get_message(self, identifier: Union[int, slice, str]) -> Union[Prompt, Response]:
+        """
+        Get a message from the session object.
+
+        :param identifier: message identifier (index or id)
```

Comment:
same

Reply:
fix in https://github.com/openscilab/memor/pull/108/commits/c53c4827ce9608a0918dfa7b5b583b76a9332918